pydarn/plotting/acf.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Marian Schmidt
+# This code is improvement based on acf.py in the DaVitpy library
+# https://github.com/vtsuperdarn/davitpy/blob/master/davitpy
+
+"""
+Range-Time Parameter (aka Intensity) plots
+"""
+import copy
+import matplotlib.pyplot as plt
+import numpy as np
+import warnings
+import matplotlib.ticker as ticker
+
+from datetime import datetime
+from typing import List
+
+from pydarn import (dmap2dict, DmapArray, DmapScalar,
+                    plot_exceptions, SuperDARNRadars,
+                    standard_warning_format, time2datetime,
+                    check_data_type)
+
+warnings.formatwarning = standard_warning_format
+
+
+class ACF():
+    """
+    ACF class plots SuperDARN RAWACF data
+
+    Class pattern design: Builder
+    This class inherits matplotlib.pyplot to inherit plotting features as well
+    build off their builder design pattern.
+
+    Notes
+    -----
+    This class extends from matplotlib to generate the figures
+
+    Methods
+    -------
+    plot
+    plot_amplitude (not implemented yet)
+    """
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - plot()\n"\
+
+    @classmethod
+    def plot(cls, dmap_data: List[dict], beam_num: int = 0,
```

Comment:
is 'plot' a bit of a general name for this?

Reply:
yea... just seemed like `pydarn.ACF.plot_acf` seemed redundant but I guess @billetd 
did it with fan plots too.